src/api/app/models/package.rb

```diff
@@ -1400,6 +1400,11 @@ def send_sysrq(params)
     backend_build_command(:sendsysrq, params[:project], params.slice(:package, :arch, :repository, :sysrq))
   end
 
+  def target_name
+    # The maintenance ID is always the sub project name of the maintenance project
+    project.is_maintenance_incident? ? "#{name}.#{project.basename}" : name
+  end
+
   def release_target_name
```

Comment:
@adrianschroeter Do you know if there is any difference between those two methods (except that release_target_name also covers old incidents)?